apps/site/lib/site_web/templates/layout/_footer.html.eex

```diff
@@ -1,107 +1,70 @@
+        <%= link "Press Releases", to: cms_static_page_path(@conn, "/news"), data: [scroll: true]  %>
+        <%= link "Policies & Civil Rights", to: cms_static_page_path(@conn, "/policies"), data: [scroll: true]  %>
+        <%= link "Safety", to: cms_static_page_path(@conn, "/safety") %>
+        <%= link "Performance", to: "http://www.mbtabackontrack.com/performance/index.html#/home", target: "_blank" %>
+      </nav>
+      <nav class="m-footer__group" aria-labelledby="footer-work-with">
+        <div id="footer-work-with" class="m-footer__group-title">Work With Us</div>
+        <%= link "Careers", to: cms_static_page_path(@conn, "/careers"), data: [scroll: true]  %>
+        <%= link "Institutional Sales", to: cms_static_page_path(@conn, "/pass-program") %>
+        <%= link "Business Opportunities", to: cms_static_page_path(@conn, "/business") %>
+        <%= link "Innovation Proposals", to: cms_static_page_path(@conn, "/innovation") %>
+        <%= link "Engineering Design Standards", to: cms_static_page_path(@conn, "/engineering/design-standards-and-guidelines") %>
+        <%= link "Developers", to: cms_static_page_path(@conn, "/developers"), data: [scroll: true] %>
+      </nav>
+    </div>
+    <div class="container">
+      <div class="lower-footer">
         <%= unless assigns[:exclude_google_translate] do %>
           <%= render "_google_translate_plugin.html", conn: @conn %>
         <% end %>
-      </li>
-    </ul>
-    <p class="m-footer__copyright">&copy; Massachusetts Bay Transportation Authority, all rights reserved.</p>
-  </div>
-</footer>
+        <div class="m-menu__language"></div>
+        <div class="social-icons-container">
+            <a href="https://www.facebook.com/TheMBTA/" target="_blank" aria-label="MBTA facebook">
+              <i class="fa-brands fa-facebook fa-xl social-icon"></i>
+            </a>
+            <a href="https://www.instagram.com/thembta/" target="_blank" aria-label="MBTA instagram">
+              <i class="fa-brands fa-instagram fa-xl social-icon"></i>
+            </a>
+            <a href="https://linkedin.com/company/mbta" target="_blank" aria-label="MBTA linkedin">
+              <i class="fa-brands fa-linkedin fa-xl social-icon"></i>
+            </a>
+            <a href="https://www.youtube.com/user/mbtagm/" target="_blank" aria-label="MBTA youtube">
+              <i class="fa-brands fa-youtube fa-xl social-icon"></i>
+            </a>
+            <a href="https://twitter.com/MBTA" target="_blank" aria-label="MBTA twitter">
+              <i class="fa-brands fa-twitter fa-xl social-icon"></i>
+            </a>
+            <a href="https://www.tiktok.com/@thembta" target="_blank" aria-label="MBTA tiktok">
+              <i class="fa-brands fa-tiktok fa-xl social-icon"></i>
+            </a>
+        </div>
+      </div>
+      <p class="m-footer__copyright">&copy; Massachusetts Bay Transportation Authority, all rights reserved.</p>
```

Comment:
I think moving this inside the `.lower-footer` element would make more sense, and also be easier to position.